reinforcement_learning/rl_managed_spot_cartpole_coach/README.md

```diff
@@ -0,0 +1,20 @@
+# Simple Cartpole Example
+
+## What is cartpole?
+
+This is a classic example of a problem that reinforcement learning can solve.  It's a simulation of balancing a broom upright by balancing it on your hand.  The broom is the "pole" and your hand is replaced with a "cart" moving back and forth on a linear track.  This simplified example works in 2 dimensions, so the cart can only move in a line back and forth, and the pole can only fall forwards or backwards, not to the sides.
+
+## This Example
+
+Sample Cart Pole Example using SageMaker RL with base docker image containing Coach, MxNet and OpenAI Gym. This is a toy example taken from 
+[Coach Quick Start Guide](https://github.com/NervanaSystems/coach/blob/master/tutorials/0.%20Quick%20Start%20Guide.ipynb). It demonstrates how you can use the `RLEstimator` from the SageMaker Python SDK in `script` mode.
```

Comment:
Maybe highlight the spot instance by adding something like 'This notebook tackles the exact same problem with the same solution, but it has been modified to be able to run using SageMaker Managed Spot infrastructure. SageMaker Managed Spot uses [EC2 Spot Instances]'